types/configs/paste-config.d.ts

```diff
@@ -31,3 +31,8 @@ export interface PasteConfig {
    */
   files?: {extensions?: string[], mimeTypes?: string[]};
 }
+
+/**
+ * Alias for PasteConfig with false
+ */
+export type PasteConfig = IPasteConfig | false;
```

Comment:
We're not using the "I" prefix with interfaces, and also the difference between `pasteConfig` and `iPasteConfig` is not clear. I'd suggest `PasteConfigSpecified` here.

```suggestion
export type PasteConfig = PasteConfigSpecified | false;
```